### ISSUE TITLE
Replace Print Statements With Log Statements

### ISSUE THREAD

**Issue by lilleswing** [deepchem/deepchem#1089]
Use the python provided logging facilities instead of print statements.

It is pretty rude we just spam stdout.

**Comment by nitinprakash96** [comment]
Hi @lilleswing,
could you give a little info on where to change these statements as the print statements could be coming from multiple files.

**Comment by lilleswing** [comment]
yes print statements come from all through the deepchem source code.

We should not have ANY print() statements inside deepchem source code.

We should replace all of them with logger.info statements.

**Comment by lilleswing** [comment]
e.x.
https://fangpenlin.com/posts/2012/08/26/good-logging-practice-in-python/

**Comment by lilleswing** [comment]
Note that a P.R. for this doesn't have to solve every issue, we can work on it gradually if the first PR sets standards that everyone can follow.

**Comment by leeyjjoel** [comment]
Hi, @nitinprakash96  and @lilleswing 

Just thought I should let you know. I've submitted a PR which replaces the print statements for deepchem/data and deepchem/dock folders. I'm currently waiting for feedback from the team to ascertain I'm making acceptable changes. @nitinprakash96 perhaps we could work together on this issue?Doing so would certainly speed up the process.

**Comment by nitinprakash96** [comment]
@leeyjjoel 
Please make sure that you let the contributor know before opening a PR or else the effort and time gets in vain.

**Comment by leeyjjoel** [comment]
Hi,

@nitinprakash96 my apologies. Wasn’t aware that you had started working on this issue. Was under the impression that you were clarifying about the nature of the task

**Comment by rbharath** [comment]
@leeyjjoel @nitinprakash96 I'm excited that you're both working hard to contribute to DeepChem. We're a team, tied together by our mutual interest in advancing the state of deep learning in science. As a result, we give collegiality and teamwork high importance. Make sure to give benefit of doubt to other team members.

We'll discuss good protocols for making sure effort isn't wasted if two contributors are interested in the same issue at the next steering committee meeting. In the meanwhile, I'd recommend opening a PR marked [WIP] if you're working on an issue. That way, other contributors can track your changes and we can avoid duplicated effort.

**Comment by leeyjjoel** [comment]
@rbharath  Alright noted. I'll close my current PR on this issue so @nitinprakash96 can try his hand at it.

@nitinprakash96  Let me know if there's anything I can help you with.

**Comment by rbharath** [comment]
@leeyjjoel Thank you for the very gracious gesture! I'm looking forward to working together to improve DeepChem and deep learning for science broadly
